docs/UserGuide.adoc

```diff
@@ -229,10 +229,50 @@ The `redo` command fails as there are no `undo` commands executed previously.
 Clears all entries from the address book. +
 Format: `clear`
 
-=== Play background music : `music`
+=== Playing background music : `music play`
 
-Plays a random music track from a list of built in mp3 music from Address Book. +
-Format: `music`
+Plays a specific genre music track from a list of built in mp3 music from Unify or resumes playing currently paused music track. +
+Format: `music play GENRE`
+
+[NOTE]
+The genre can only be either pop, dance or classic
```

Comment:
The genre can only be ~either~ pop, dance or classic. Either is for 2 options only.

Maybe we can add a note about the functionality of `music play`?